public/app/features/alerting/unified/components/rule-viewer/AlertRuleMenu.tsx

```diff
@@ -1,6 +1,7 @@
 import { AppEvents } from '@grafana/data';
 import { ComponentSize, Dropdown, Menu } from '@grafana/ui';
 import appEvents from 'app/core/app_events';
+import { t } from 'app/core/internationalization';
```

Comment:
changes in this file are only about translations (this is because i added some changes for deleting rule permanently that i finally reverted)